src/journey-maps/package.json

```
@@ -13,6 +13,12 @@
     "url": "https://github.com/sbb-design-systems/sbb-angular/issues"
   },
   "homepage": "https://angular.app.sbb.ch",
+  "peerDependencies": {
+    "@angular/common": "0.0.0-NG",
+    "@angular/core": "0.0.0-NG",
+    "@types/geojson": "7946",
+    "maplibre-gl": "2.1"
```

Should we maybe use a range? Or would you prefer more precise versions?
```suggestion
    "@types/geojson": ">=7946",
    "maplibre-gl": "^2.1.0"
```

---

👍